src/app/services/ErrorLoggerService.ts

```diff
@@ -1,7 +1,7 @@
 import { Injectable } from '@angular/core';
-import { AngularFirestore } from '@angular/fire/firestore';
+import { Firestore } from '@angular/fire/firestore';
```

Comment:
So what is this difference and why ?

Reply:
pas de différence, c'est l'upgrade vers la nouvelle version d'AngularFire (ils ont refait l'API)